config/deploy/docker/assets/entrypoint.sh

```diff
@@ -63,4 +63,4 @@ cd ../..
 
 # Then exec the container's main process (what's set as CMD in the Dockerfile).
 echo "calling: $@"
-exec "$@"
+bundle exec "$@"
```

Comment:
Did you test this? the running process in the container will now be in a subprocess.

Reply:
I couldn't get it to work without that.  I think the issue is that `$@` is `rails asset:compile` or something similar, which needed the bundle.  I suppose we could change what gets passed in to include the `bundle`

Reply:
I'd try changing that to `./bin/rails asset:compile` instead if possible. Though, if this doesn't break anything it's fine as is.